tests/test_browser.py

```diff
@@ -1767,6 +1767,9 @@ def test_cubegeom_pre2_vao(self):
   def test_cubegeom_pre2_vao2(self):
     self.btest('cubegeom_pre2_vao2.c', reference='cubegeom_pre2_vao2.png', args=['-s', 'LEGACY_GL_EMULATION=1', '-lGL', '-lSDL'])
 
+  def test_cubegeom_pre_vao_es(self):
+    self.btest('cubegeom_pre_vao_es.c', reference='cubegeom_pre_vao.png', args=['-s', 'FULL_ES2=1', '-lGL', '-lSDL'])
```

Comment:
This new test fails on Firefox on OS X, though does not look like it would be the fault of the test though. See https://github.com/kripken/emscripten/issues/5408